backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceHeaderImplBuilder.java

```diff
@@ -715,4 +726,34 @@ public void setC_PaymentInstruction_ID(final int c_PaymentInstruction_ID)
 		C_PaymentInstruction_ID = c_PaymentInstruction_ID;
 	}
 
+
+	public int getC_Harvesting_Calendar_ID()
+	{
+		return C_Harvesting_Calendar_ID;
+	}
+
+	public void setC_Harvesting_Calendar_ID(final int calendarId)
+	{
+		C_Harvesting_Calendar_ID = calendarId;
```

Comment:
now it seems to me that if there are multiple ICs with different calendarIds, then what every ICs happens to be the last one aggregated wins.
What *we* want is

> , the resulting invoice shall have a `null` `C_Harvesting_Calendar_ID`.

See `/InvoiceHeaderImplBuilder#setSalesRep_ID` for a working example

Reply:
thx for the hint. tested the changes and pushed the code changes